cli.go

```diff
@@ -8,11 +8,15 @@ import (
 	"log"
 	"os"
 	"os/signal"
+	"strings"
 	"sync"
 	"syscall"
 	"time"
 
+	mcli "github.com/mitchellh/cli"
+
```

Comment:
Not sure if this is intentional to have this import grouped by itself

Reply:
Not intentional, I usually add the an import somewhere and assumed goimports organized it nicely. Looks like goimports respects newlines. I'll move it to be with the other non-standard library imports.